packages/mutation-testing-elements/src/style/prismjs.scss

```diff
@@ -1,4 +1,5 @@
 
 
 @import "./prismjs.css";
-@import "~prismjs/plugins/line-numbers/prism-line-numbers.css";
+// @import "~prismjs/plugins/line-numbers/prism-line-numbers.css";
```

Comment:
Why is this commented out?

Reply:
Good catch! I was playing around with prism diff's (which are incompatible with the line numbers plugin). 😅